src/project_template/cogs/error_handler.py

```diff
@@ -0,0 +1,23 @@
+import discord
+from discord.ext import commands
+
+from project_template import config
+from project_template.utils.logger import log
+
+
+class ErrorHandler(commands.Cog):
+    @commands.Cog.listener()
+    async def on_command_error(self, ctx: commands.Context, error: commands.CommandError):
+        """A global command error handler."""
+        errorName = str(error).replace("Command raised an exception: ", "")
```

Comment:
See if you can't use `error.args` to get the name without string replacement.

Also use `traceback.format_exc()` to get the full exception, in addition to the name. The name of the exception isn't as useful without the stack trace, so we'll want both.

Reply:
OK, I'll try that soon